### PR TITLE
Remove 52lts compatiblity layer in v2v

### PR DESCRIPTION
**Pull request by xiaodwan**
Signed-off-by: Xiaodai Wang <xiaodwan@redhat.com>

### CODE REVIEW

**Comment by xiaodwan** [comment]
Depend on https://github.com/avocado-framework/avocado-vt/pull/2450

**Comment by xiaodwan** [comment]
(1/3) type_specific.io-github-autotest-libvirt.convert_vm_to_libvirt.esx.vm.6_7.linux.latest8.arch_x86_64.it_vddk: PASS (2339.95 s)
(1/2) type_specific.io-github-autotest-libvirt.convert_from_file.positive_test.windows.input_mode.vmx.vmx_default.win2019.output_mode.libvirt: PASS (1092.91 s)
(1/1) type_specific.io-github-autotest-libvirt.v2v_options.positive_test.vmx_ssh.input_mode.none.output_mode.libvirt: PASS (603.02 s)

**Comment by xiaodwan** [comment]
@chunfuwen @kylazhang @chloerh  Please help to review this PR, thanks.

File: v2v/tests/src/v2v_options.py [hunk 1]
```diff
@@ -11,6 +11,7 @@
 
 from avocado.core import exceptions
 from avocado.utils import process
+from avocado.utils.astring import to_text
```
Comment (chunfuwen) [inline v2v/tests/src/v2v_options.py]: not just rename module, but check each place to use it.
For example, one place use as below :
         ret = to_text(process.system_output("grep -R \"%s\" %s" %
                                                (ovf_id, export_vm_dir)))

Better replace :
procces.run().stdout_text directly, see https://docs.google.com/document/d/1Zq6SZ1kduxK9odgOtLxKW0D0Fhp8zzh5yxgh1HVmfzY/edit

Comment (xiaodwan) [reply v2v/tests/src/v2v_options.py]: > not just rename module, but check each place to use it.
> For example, one place use as below :
> ret = to_text(process.system_output("grep -R "%s" %s" %
> (ovf_id, export_vm_dir)))
> 
> Better replace with:
> procces.run().stdout_text directly, see https://docs.google.com/document/d/1Zq6SZ1kduxK9odgOtLxKW0D0Fhp8zzh5yxgh1HVmfzY/edit

I inserted a code review comments and didn't submit it. so you may not see it.
Here v2v uses an alias in old codes, only update the import line is enough.

**Pull request merged**